app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt

```diff
@@ -37,15 +41,21 @@ private const val BUNDLE_CHAT_ROOM_ID = "chat_room_id"
 private const val BUNDLE_CHAT_ROOM_NAME = "chat_room_name"
 private const val BUNDLE_CHAT_ROOM_TYPE = "chat_room_type"
 private const val BUNDLE_IS_CHAT_ROOM_READ_ONLY = "is_chat_room_read_only"
+private const val REQUEST_CODE_FOR_PERFORM_SAF = 42
 
 class ChatRoomFragment : Fragment(), ChatRoomView {
-
     @Inject lateinit var presenter: ChatRoomPresenter
     private lateinit var chatRoomId: String
     private lateinit var chatRoomName: String
     private lateinit var chatRoomType: String
     private var isChatRoomReadOnly: Boolean = false
     private lateinit var adapter: ChatRoomAdapter
+    // For reveal and unreveal anim.
+    private val hypotenuse by lazy { Math.hypot(relative_layout.width.toDouble(), relative_layout.height.toDouble()).toFloat() }
```

Comment:
Maybe use a custom view and have an `attach` method passing the anchor view. Also, `relative_layout` is not very descriptive